_inc/client/plans/product-selector.jsx

```diff
@@ -141,12 +151,34 @@ class ProductSelector extends Component {
 		return false;
 	}
 
+	handleLandingPageLinkClick = () => {
+		analytics.tracks.recordJetpackClick( {
+			target: 'landing-page-link',
+			feature: 'single-product-backup',
+			extra: this.state.selectedBackupType,
+		} );
+	};
+
+	setSelectedBackupType = selectedBackupType => {
+		this.setState( { selectedBackupType } );
+	};
+
 	renderTitleSection() {
 		return (
 			<Fragment>
 				<h1 className="plans-section__header">{ __( 'Solutions' ) }</h1>
 				<h2 className="plans-section__subheader">
 					{ __( "Just looking for backups? We've got you covered." ) }
+					<br />
+					<ExternalLink
+						target="_blank"
+						href="https://jetpack.com/upgrade/backup/"
```

Comment:
Should we use `getUpgradeUrl` here, so we get the different source query strings added to the URL so we can track things on Jetpack.com and redirect to the right Calypso page afterwards?

Reply:
@jeherve I don't think so. The `getUpgradeUrl` brings a user to Calypso where they can go through the checkout flow.

This URL (https://jetpack.com/upgrade/backup/) brings the user directly to the landing page where more information can be found around available backup options (our desired destination).

We do the same in Calypso: https://github.com/Automattic/wp-calypso/pull/38311/files#diff-67e84bb8c75685ed38b07fe2d8d7dd74R104

Is it possible to land on https://jetpack.com/upgrade/backup/ using `getUpgradeUrl`?

Reply:
> Is it possible to land on jetpack.com/upgrade/backup using getUpgradeUrl?

Yes, you decide where it lands from WordPress.com, thanks to the Jetpack Redirects lib.

Reply:
I've switched it up to use `aag-backup` in ad0da4a8326410a0a6f2aa2a1d09d6e561a1ca26, and it seems to land me on the correct spot in my local testing. Can y'all confirm this is the correct redirect code?

Reply:
This seems correct. Thank you for that, @ChaosExAnima!